### ISSUE TITLE
Having tempory files like .#vagrantfile# or #vagrantfile# we fail VM startup during /vagrant sync

### ISSUE THREAD

**Issue by maxandersen** [projectatomic/adb-atomic-developer-bundle#275]
seems rsync on windows fails if you have files like #Vagrantfile# which vi leaves behind.

![desktop-738r19v](https://cloud.githubusercontent.com/assets/54129/13543560/a628db74-e26c-11e5-84ed-3e38ca33f23f.png)

can we not guard against this since things are not working after this and I have to manually remove the file and reprovision afaics.

**Comment by praveenkumar** [comment]
@maxandersen vim by default have back file option set iirc. if you unset it from vimrc then it will not save any backup file or even better you can change the path to those temp file.

Change in vimrc file

```
set backupdir=~/vimtmp,.
set directory=~/vimtmp,.
```

**Comment by maxandersen** [comment]
Yes, I know I can turn it off in vim, but that doesn't help the users that ends up in this situation.

**Comment by hferentschik** [comment]
Not sure there is much we can do tbh. I think the best option is still to disable the /vagrant synced folder or at least not force it to rsync - https://github.com/projectatomic/adb-atomic-developer-bundle/issues/272
